frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/MfaResetAuthorizeHandlerTest.java

```diff
@@ -0,0 +1,170 @@
+import static uk.gov.di.authentication.sharedtest.matchers.APIGatewayProxyResponseEventMatcher.hasStatus;
+
+class MfaResetAuthorizeHandlerTest {
+    private static final SerializationService objectMapper = SerializationService.getInstance();
+    private static final ConfigurationService configurationService =
+            mock(ConfigurationService.class);
+    private final MfaResetIPVAuthorizationService mfaResetIPVAuthorizationService =
+            mock(MfaResetIPVAuthorizationService.class);
+    private static final AuthenticationService authenticationService =
+            mock(AuthenticationService.class);
+    private static final ClientSessionService clientSessionService =
+            mock(ClientSessionService.class);
+    private static final ClientService clientService = mock(ClientService.class);
+    private static final Context context = mock(Context.class);
+    private static final SessionService sessionService = mock(SessionService.class);
+    private static final UserContext userContext = mock(UserContext.class);
+    private static final Session session = mock(Session.class);
+    private static final AuditContext testAuditContext =
+            new AuditContext(
+                    AuditService.UNKNOWN,
+                    CLIENT_SESSION_ID,
+                    SESSION_ID,
+                    AuditService.UNKNOWN,
+                    EMAIL,
+                    IP_ADDRESS,
+                    AuditService.UNKNOWN,
+                    DI_PERSISTENT_SESSION_ID,
+                    Optional.of(ENCODED_DEVICE_DETAILS));
+    private static final APIGatewayProxyRequestEvent TEST_INVOKE_EVENT =
+            ApiGatewayProxyRequestHelper.apiRequestEventWithHeadersAndBody(
+                    CommonTestVariables.VALID_HEADERS,
+                    format("{ \"email\": \"%s\" }", EMAIL.toUpperCase()));
+    private static MfaResetAuthorizeHandler handler;
+
+    @BeforeAll
+    static void globalSetup() {
+        when(userContext.getSession()).thenReturn(new Session(SESSION_ID));
+        when(userContext.getClientSessionId()).thenReturn(CLIENT_SESSION_ID);
+        when(session.getInternalCommonSubjectIdentifier()).thenReturn(COMMON_SUBJECT_ID);
+        when(sessionService.getSessionFromRequestHeaders(anyMap()))
+                .thenReturn(Optional.of(session));
+        when(session.getSessionId()).thenReturn(SESSION_ID);
+    }
+
+    @BeforeEach
+    void testSetup() {
+        handler =
+                new MfaResetAuthorizeHandler(
+                        configurationService,
+                        sessionService,
+                        clientSessionService,
+                        clientService,
+                        authenticationService,
+                        mfaResetIPVAuthorizationService);
+    }
+
+    @Test
+    void returnsA200WithRedirectUriInBody() throws Json.JsonException {
+        String TEST_REDIRECT_URI = "https://some.uri.gov.uk/authorize";
```

Comment:
If you want to make this a constant (which is what the naming implies) then you need a `final` before the `String`

Reply:
which is what sonar is moaning about